opendevin/action/fileop.py

```diff
@@ -69,14 +69,17 @@ async def run(self, controller) -> Observation:
             read_lines = self._read_lines(content.split('\n'))
             code_view = ''.join(read_lines)
         else:
-            whole_path = resolve_path(self.path)
-            self.start = max(self.start, 0)
             try:
-                with open(whole_path, 'r', encoding='utf-8') as file:
-                    read_lines = self._read_lines(file.readlines())
-                    code_view = ''.join(read_lines)
-            except FileNotFoundError:
-                return AgentErrorObservation(f'File not found: {self.path}')
+                whole_path = resolve_path(self.path)
+                self.start = max(self.start, 0)
+                try:
+                    with open(whole_path, 'r', encoding='utf-8') as file:
+                        read_lines = self._read_lines(file.readlines())
+                        code_view = ''.join(read_lines)
+                except FileNotFoundError:
+                    return AgentErrorObservation(f'File not found: {self.path}')
+            except PermissionError:
+                return AgentErrorObservation(f'Malformed paths not permitted: {self.path}')
```

Comment:
```suggestion
                return AgentErrorObservation(f'Malformed paths not permitted: {self.path}')

```

just triggering a push to test GH actions